src/wallet/wallet.cpp

```diff
@@ -4641,7 +4654,7 @@ bool CWallet::InitLoadWallet(bool clearWitnessCaches)
 
     RegisterValidationInterface(walletInstance);
 
-    CBlockIndex *pindexRescan = chainActive.Tip();
+    CBlockIndex *pindexRescan;
```

Comment:
Gratuitous change, this variable doesn't need to be initialized.

Reply:
I'm always suspicious of uninitialized variables, because they're fragile in refactoring. Unfortunately `if` isn't an expression in C++, but maybe we could initialize this to `chainActive.Genesis()` here and then only reset it in the successful `walletdb.ReadBestBlock` case below?